src/app/static/src/app/components/header/OnlineSupportMenu.vue

```diff
@@ -10,13 +10,14 @@
                @click="toggleErrorReportModal"
                v-translate="'troubleshootingRequest'">
             </a>
-            <router-link id="accessibility-link"
-                         to="/accessibility"
-                         class="dropdown-item"
-                         v-translate="'axe'">
-            </router-link>
+            <a class="dropdown-item"
+               id="accessibility-link"
+               :href="'/accessibility'"
+               target="_blank"
+               v-translate="'axe'">
+            </a>
         </drop-down>
-        <error-report :open="errorReportOpen" @close="toggleErrorReportModal"></error-report>
+        <error-report v-if="show" :open="errorReportOpen" @close="toggleErrorReportModal"></error-report>
```

Comment:
Have introduced the router [here](https://github.com/mrc-ide/hint/pull/623) so accessibility should remain a `router-link`.

But I don't think this is the right approach here anyway. This component should simply not be included in the new header, and a new component created for the data exploration mode, since it will need it's own error report component and presumably it's own FAQ at some point. This should be a separate ticket.